storybookjs/storybook
# Angular: Fix error handling for angular builder standalone builds

Issue: https://github.com/storybookjs/storybook/issues/15555

## What I did


Duplicate code for Standalone error handling from https://github.com/storybookjs/storybook/blob/39c7ba09ad84fbd466f9c25d5b92791a5450b9f6/lib/core-server/src/build-dev.ts#L136
and add it for angular builder

## How to test

add bad import in angular-cli examples 
for ex :
```
import { aa } from 'aaaa/aaa';
``` 
in `examples/angular-cli/src/stories/welcome-angular.stories.ts`

see error on console with this cmd : `yarn ng run angular-cli:storybook` or `yarn ng run angular-cli:build-storybook`


- [x] Is this testable with Jest or Chromatic screenshots? na
- [x] Does this need a new example in the kitchen sink apps? na
- [x] Does this need an update to the documentation? na
